tests/validator/validator.cpp

```diff
@@ -25,6 +25,14 @@ limitations under the License.
  * are not picked up by the main tests testing the API of the library
  */
 
+TEST(Validator, invalidInput)
+{
+    libcellml::ValidatorPtr validator = libcellml::Validator::create();
+
+    validator->validateModel(nullptr);
+    EXPECT_EQ(size_t(1), validator->issueCount());
```

Comment:
should we also be testing the error message since this is a new check?

Reply:
7b0b90c